src/acquisition/norostat/norostat_add_history.py

```diff
@@ -18,28 +18,42 @@
 from . import norostat_raw
 
 
-
 def main():
-  norostat_sql.ensure_tables_exist()
-  snapshot_dir = os.path.expanduser("~/norostat_history/wayback/websites/www.cdc.gov/norovirus/reporting/norostat/data-table.html/")
-  snapshot_version_counter = collections.Counter()
-  for subdir in os.listdir(snapshot_dir):
-    if re.match(r'[0-9]+', subdir) is not None:
-      # appears to be snapshot dir
-      snapshot_version_counter[subdir] = 0 # register that loop found this snapshot directory
-      for norostat_capitalization in ["norostat","noroSTAT"]:
-        time.sleep(0.002) # ensure parse times are unique, assuming OS can accurately sleep and measure to ms precision
-        path = os.path.join(snapshot_dir,subdir,"norovirus","reporting",norostat_capitalization,"data-table.html")
-        if os.path.isfile(path):
-          print("Processing file ", path)
-          with open(path, 'r') as datatable_file:
-            content = datatable_file.read()
-          wide_raw = norostat_raw.parse_content_to_wide_raw(content)
-          long_raw = norostat_raw.melt_wide_raw_to_long_raw(wide_raw)
-          norostat_sql.record_long_raw(long_raw)
-          snapshot_version_counter[subdir] += 1
-  print('Successfully uploaded the following snapshots, with the count indicating the number of data-table versions found inside each snapshot (expected to be 1, or maybe 2 if there was a change in capitalization; 0 indicates the NoroSTAT page was not found within a snapshot directory); just "Counter()" indicates no snapshot directories were found:', snapshot_version_counter)
-  norostat_sql.update_point()
+    norostat_sql.ensure_tables_exist()
+    snapshot_dir = os.path.expanduser(
+        "~/norostat_history/wayback/websites/www.cdc.gov/norovirus/reporting/norostat/data-table.html/"
+    )
+    snapshot_version_counter = collections.Counter()
+    for subdir in os.listdir(snapshot_dir):
+        if re.match(r"[0-9]+", subdir) is not None:
+            # appears to be snapshot dir
+            snapshot_version_counter[subdir] = 0  # register that loop found this snapshot directory
+            for norostat_capitalization in ["norostat", "noroSTAT"]:
+                time.sleep(
+                    0.002
+                )  # ensure parse times are unique, assuming OS can accurately sleep and measure to ms precision
```

Comment:
hm, i dont like that it's taking the inline comment into account for line wrapping